CSS/utilities.css

```diff
@@ -37,6 +41,9 @@ body{
     /* --text-color:#e5e7eb ; */
 
     --primary-color: #6B63FF;
+    --danger-color : #F44336;
+    --success-color :#4CAF50;
+    --warning-color : #FF9800;
```

Comment:
define colors in different variables folder, and call the variables in utilities